examples/map/map_metadata_modification.py

```diff
@@ -0,0 +1,29 @@
+"""
+=========================
+Map metadata modification
+=========================
+
+How to query map metadata for changes.
+
+sunpy has a series of map sources that can fix common issues with the original metadata
+stored in FITS files. A copy of the original (unaltered) metadata is stored, so
+any changes that sunpy (or the user) subsequently makes to the metadata can be
+easily queried.
+
+In the example below, we load a HMI sample image, and query the metadata for
+any added, removed, or modified items.
+"""
+import sunpy.map
+from sunpy.data.sample import HMI_LOS_IMAGE
+
+###############################################################################
+# Start by creating a map
+hmimap = sunpy.map.Map(HMI_LOS_IMAGE)
```

Comment:
`SWAP_LEVEL1_IMAGE` might be a better example since it has an added item.  (Although _why_ it adds `obsrvtry` I don't know...)

Reply:
I stuck with this map, but rotated it which demonstrates all three new meta properties.